docs/docsite/rst/roadmap/COLLECTIONS_4.rst

```diff
@@ -12,7 +12,7 @@ Release schedule
 =================
 
 
-:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4.
+:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. Request to include new collection can be submitted `here <https://github.com/ansible-collections/ansible-inclusion>`_.
```

Comment:
We should avoid using non-descriptive link titles such as `here`. Maybe this is better:
```suggestion
:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. `Requests to include new collections can be submitted <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
```
(I also adjusted the link to directly create a new discussion.)